updater/Gemfile.lock

```diff
@@ -127,19 +127,19 @@ GEM
       public_suffix (>= 2.0.2, < 6.0)
     ast (2.4.2)
     aws-eventstream (1.2.0)
-    aws-partitions (1.624.0)
+    aws-partitions (1.644.0)
     aws-sdk-codecommit (1.51.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-core (3.137.0)
+    aws-sdk-core (3.159.0)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.525.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1, >= 1.6.1)
     aws-sdk-ecr (1.56.0)
       aws-sdk-core (~> 3, >= 3.127.0)
       aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.5.1)
+    aws-sigv4 (1.5.2)
```

Comment:
I'm unclear why Dependabot hasn't opened a PR to bump these? 
https://github.com/dependabot/dependabot-core/pulls?q=is%3Apr+aws+author%3Aapp%2Fdependabot+

Maybe it's because we've got Dependabot setup to watch this as a library rather than an app? But for `/updater` it should be treating it as an app and bumping these... Need to investigate...

Reply:
I think it's because dependabot does not update indirect dependencies by default?